app/code/Magento/Security/Model/Config.php

```diff
@@ -26,10 +26,17 @@ class Config implements ConfigInterface
 
     /**
      * Configuration path to fronted area
+     * @deprecated
+     * @see \Magento\Security\Model\Config::XML_PATH_FRONTEND_AREA
      */
     const XML_PATH_FRONTED_AREA = 'customer/password/';
```

Comment:
I would also suggest using `self::XML_PATH_FRONTEND_AREA` as the value then we will not need to maintain two versions of the constant.

Reply:
@dmanners Done